modules/localization/proto/imu.proto

```diff
@@ -5,7 +5,7 @@ package apollo.localization;
 import "modules/common/proto/header.proto";
 import "modules/localization/proto/pose.proto";
 
-message Imu {
```

Comment:
Do you want to change the name or just deprecate this one and add a new message instead? This would cause inconsistency in recorded data right?